config/pitest-suppressions/pitest-javadoc-suppressions.xml

```diff
@@ -855,14 +855,14 @@
     <lineContent>while (count &lt;= 1</lineContent>
   </mutation>
 
-  <mutation unstable="false">
-    <sourceFile>SummaryJavadocCheck.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck</mutatedClass>
-    <mutatedMethod>containsForbiddenFragment</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.NakedReceiverMutator</mutator>
-    <description>replaced call to java/lang/String::trim with receiver</description>
-    <lineContent>.matcher(firstSentence).replaceAll(&quot; &quot;).trim();</lineContent>
-  </mutation>
+
+
+
+
+
+
+
```

Comment:
Please remove the empty lines.

Reply:
It was their to avoid conflicts

Reply:
I am ok if you want to remove these at the end of the mutation killing. Resolved.